app/Http/Controllers/Bible/BooksControllerV2.php

```diff
@@ -104,7 +104,9 @@ function () use ($id, $fileset, $testament) {
                     $books[$key]->bible_id        = $bible_id;
                     $books[$key]->number_chapters = $current_chapters[$key]->count();
                     $books[$key]->chapters        = $current_chapters[$key]->implode(',');
-                    $books[$key]->name            = $book_translation[$book->id];
+                    $books[$key]->name            = isset($book_translation[$book->id])
+                        ? $book_translation[$book->id]
+                        : '';
```

Comment:
Related with the issue:

```log
Exception 'ErrorException' with message 'Undefined array key "GEN"' in /var/app/current/vendor/laravel/framework/src/Illuminate/Collections/Collection.php:1595
```
It will make sure the `book_id` is indexed into `book_translation` array